### ISSUE TITLE
unexpected aria-expanded paragraph in tablist def

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#1402]
in the ARIA 1.2 spec `tablist` has a paragraph of content that states (bolded text starting with "each visible tabpanel" and ending with the end of the quote, mine for emphasis)

>For a single-selectable tablist, authors SHOULD hide other tabpanelelements from the user until the user selects the tab associated with that tabpanel. For a multi-selectable tablist, authors SHOULD ensure **each visible tabpanel has its aria-expandedattribute set to true, and that the remaining hidden tabpanel elements have their aria-expanded attributes set to false**.

But `tabpanel` doesn't have `aria-expanded` as a supported attribute anymore... and this doesn't really make sense for `aria-expanded` to be on the `tabpanel` anyway.

the `tab` role has a very similar paragraph but it states that the `tab` is the element that SHOULD have `aria-expanded` set to `true/false`.  And that makes a lot more sense.

Seems to me this issue can be resolved by aligning the `tablist` content to match the `tab`content.

**Comment by jnurthen** [comment]
I think this is a dup of https://github.com/w3c/aria/issues/1032

**Comment by carmacleod** [comment]
@jnurthen 
Yes, this resolves #1032 as well.
Thanks for volunteering to review... I was coincidentally creating PR #1402 as you were typing. ;)
